common/hello.go

```diff
@@ -0,0 +1,103 @@
+package common
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"path"
+	"time"
+
+	"github.com/giantswarm/apiextensions-application/api/v1alpha1"
+	"github.com/giantswarm/clustertest/pkg/application"
+	"github.com/giantswarm/clustertest/pkg/organization"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/giantswarm/cluster-test-suites/internal/state"
+)
+
+func runHelloWorld() {
+	Context("hello world", func() {
+		var nginxApp, helloApp *v1alpha1.App
+		var nginxConfigMap, helloConfigMap *v1.ConfigMap
+
+		BeforeEach(func() {
+			ctx := context.Background()
+			org := organization.New("giantswarm")
+
+			nginxApp, nginxConfigMap = deployApp(ctx, "ingress-nginx", "kube-system", org, "3.0.0", "./test_data/nginx_values.yaml")
+			Eventually(func() error {
+				app, err := state.GetFramework().GetApp(ctx, nginxApp.Name, nginxApp.Namespace)
+				if err != nil {
+					return err
+				}
+				return checkAppStatus(app)
+			}).
+				WithTimeout(3 * time.Minute).
+				WithPolling(5 * time.Second).
+				Should(Succeed())
+
+			helloApp, helloConfigMap = deployApp(ctx, "hello-world", "giantswarm", org, "2.0.0", "./test_data/helloworld_values.yaml")
+			Eventually(func() error {
+				app, err := state.GetFramework().GetApp(ctx, helloApp.Name, helloApp.Namespace)
+				if err != nil {
+					return err
+				}
+				return checkAppStatus(app)
+			}).
+				WithTimeout(3 * time.Minute).
+				WithPolling(5 * time.Second).
+				Should(Succeed())
+		})
+
+		It("hello world app responds successfully", func() {
+			Eventually(func() (*http.Response, error) {
+				return http.Get(fmt.Sprintf("https://hello-world.%s.gaws.gigantic.io", state.GetCluster().Name))
```

Comment:
could we read the domain from somewhere like from cluster-values CM?

Reply:
Same in `providers/capa/standard/test_data/helloworld_values.yaml`

Reply:
Done